src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix.motorcontrol.can.BaseMotorController; 
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX; // internal Falcon motors
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+
+import edu.wpi.first.wpilibj.XboxController; // controller input 
+
+import com.revrobotics.CANSparkMax; // front intake motor
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import com.revrobotics.CANSparkMax.IdleMode;
+
+import edu.wpi.first.wpilibj.Servo;
+
+import static frc.robot.Constants.IntakeConstants.*;
+
+
+public class Intake extends SubsystemBase {
+    
+    private final CANSparkMax front = new CANSparkMax(FRONT_MOTOR, MotorType.kBrushless); // front motor for intake
```

Comment:
Use `MotorUtil.createSparkMax(int)`? Same with the other `TalonSRX` constructors; using the util lets you not have to worry about factory resetting, since the method automatically does that and applies global config settings.